summit.proto

```diff
@@ -855,4 +855,48 @@ message AdaptiveState
 	double Prog2AmpInMilliamps = 3;
 	double Prog3AmpInMilliamps = 4;
 	double RateTargetInHz = 5;

```

Comment:
it might be nice to pass back IsValid, if indeed error checking is happening on the device. this is something we could ask Jeff about, as he would be familiar with it. one of the properties of protobufs seems to be that message fields are not explicitly required. so if we added isValid here, it could be populated during read operations, but our end users (if they understood this point) wouldn't be required to provide that field in a write operation, where it wouldn't really make sense to provide it.